### DISCUSSION TITLE
OfflinePacks only for mapbox:// url scheme?

### DISCUSSION THREAD

**Discussion by 386sx** [rnmapbox/maps#2545] [Q&A]
Hi everyone,

I've been using rnmapbox in many years now and we are rewriting stuff for v10. 
We host an own vector tile server and everything worked fine until now. I ask here, because I don't know if this is a bug or a feature:
- since some time it is just possible to download tiles with a mapbox url scheme like `mapbox://styles/mapbox/satellite-streets-v10` but not https://my-vector-tile-serve/style.json.

Is this expected behaviour or should I file an issue?

**Comment by Cdingram** [comment]
I have same issue

**Comment by mfazekas** [comment]
We also have https://rnmapbox.github.io/docs/components/offlineManagerLegacy that builds on OfflineRegionManager that should handle non mapbox url-s as well

**Comment by Cdingram** [comment]
Can confirm legacy offline manager does work with non-mapbox urls. Snapshot manager has the same issue with non-mapbox urls though.
